s3-eventbridge-ecs/README.md

```diff
@@ -0,0 +1,100 @@
+# Serverless pattern to integrate EventBridge and ECS-Fargate tasks
+
+![Serverless Pattern Architecture](s3-eb-ecs.png)
+
+This serverless pattern uses EventBridge to trigger multiple ECS tasks on Fargate. The ECS task queries the S3 bucket for files and reads them. These ECS tasks can be extended to insert the data to a database, thus speeding up the process of loading data from S3 to a database.
+
+This application pattern can be extended to
+
+    1. Process a file from a single bucket in parallel
+    2. Perform long running tasks not suited for Lambda processing in parallel
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+# Requirements
+
+- [Create an AWS account](https://aws.amazon.com) if you do not already have, create them and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Terramform installed](https://www.terraform.io/)
+
+# Deployment Instructions
+
+1. Configure AWS CLI to the account and region you want to deploy the application to
+
+2. Create a new directory then navigate to that directory in a terminal and clone the GitHub repository:
+    ~~~ code
+    git clone https://github.com/aws-samples/serverless-patterns
+    
+3. Change directory to the pattern directory:
+    ~~~ code
+    cd serverless-patterns/s3-eventbridge-ecs
+    
+4. Go to the [Docker directory](src/docker/README.md) and execute the docker command as per the Readme file. The docker commands in the file will perform the following:
+    - It will create an ECR Repository in your AWS account in the preferred AWS region
+    - It will create the task docker image and push it to the ECR repository
+    - Once done, login to the AWS console and search for ECR. Note the ARNs of the image name for the task. The ARN and image name will be needed in terraform script
+
+    ![ECR Image ARN](ecr-task-uri.JPG)
+
+5. Make the following changes in the [terraform script](pattern_s3_eb_ecs.tf):
+    - Line 2 - Replace the placeholder "REPLACE_ME_WITH_S3_BUCKET" with an unique S3 bucket name. you can create an S3 bucket with CLI commnand as below :
+        ~~~ code
```

Comment:
why this cannot be created as Iac?

Reply:
The terraform will create the S3. Users just need to provide an unique S3 bucket name.